source/tutorial/swiftui.txt

```diff
@@ -21,12 +21,20 @@ the creation of a to-do list application. This application enables users to:
 
 - Register their email as a new user account.
 - Sign in to their account with their email and password (and sign out later).
-- View, create, modify, and delete tasks.
+- View, create, modify, and delete their own tasks.
+- View all tasks, even where the user is not the owner.
 
-This tutorial adds on to the Template App. You will add a new ``priority`` field 
+The template app also provides a toggle that simulates the device being in 
+"Offline Mode." This toggle lets you quickly test Device Sync functionality 
+on a simulator, simulating the user having no internet connection. However,
```

Comment:
```suggestion
on a simulator, simulating the user having no internet connection. However,
```

"simulator, simulating" reads odd to me. perhaps 'simulating' could be replaced by a synonym like 'mocking', 'emulating', etc.